README.md

```diff
@@ -9,11 +9,23 @@ client-id: ${CLIENT_ID}
 client-secret: ${CLIENT_SECRET}
 project-key: ${PROJECT_KEY}
 ```
+## Docker set up (sbt-native-packager)
+Create a local docker image:
+```
+# sbt docker:publishLocal
+```
+Run the image with the environment variables defined either in a file:
+```
+# docker run -it --env-file <.env file> <image ID>
+```
+or explicitly:
+```
+# docker run -it -e <VAR=VAL> -e <VAR=VAL> <image ID>
```

Comment:
Can you add an example with which environment variables are expected?

Reply:
sure, will add

Reply:
done